classes/setup_form.class.php

```diff
@@ -165,7 +165,8 @@ public function save($data) {
         $turnitinviewermatchsubinfo = (!empty($data->permissionoptions['turnitinviewermatchsubinfo'])) ? 1 : 0;
         $turnitinviewersavechanges = (!empty($data->permissionoptions['turnitinviewersavechanges'])) ? 1 : 0;
 
-        set_config('turnitinsim_use', $useplugin, 'plagiarism');
+        plagiarism_plugin_turnitinsim::enable_plugin($useplugin);
```

Comment:
As in `db/upgrade.php` we have logic to use old way if the version is less than 2020061201, do we need some type of logic here as well or this line will work in all versions?